### PR TITLE
Message payload

### PR DESCRIPTION
**Pull request by Misiu**
This allows specifying message payload.
When specified **all messages** will be sent with this payload.

Please notice this is my first try on Go lang.

### CODE REVIEW

File: client.go [hunk 1]
```diff
@@ -70,13 +70,26 @@ func (c *Client) Run(res chan *RunResults) {
 }
 
 func (c *Client) genMessages(ch chan *Message, done chan bool) {
-	for i := 0; i < c.MsgCount; i++ {
-		ch <- &Message{
-			Topic:   c.MsgTopic,
-			QoS:     c.MsgQoS,
-			Payload: make([]byte, c.MsgSize),
+	if c.MsgPayload != "" {
```
Comment (krylovsk) [inline client.go:73]: What about refactoring that to use `switch/case` ?

File: main.go [hunk 2]
```diff
@@ -60,6 +60,7 @@ func main() {
 	var (
 		broker   = flag.String("broker", "tcp://localhost:1883", "MQTT broker endpoint as scheme://host:port")
 		topic    = flag.String("topic", "/test", "MQTT topic for outgoing messages")
+		payload  = flag.String("payload", "", "MQTT message payload. If empty, then payload is generated based on the size parameter")
```
Comment (krylovsk) [inline main.go:63]: Please add an example running the tool with a custom payload in README.

File: client.go (hunk 1 above)
Comment (Misiu) [reply client.go:73]: Not sure is switch/case is needed here. I only need two cases - MSGPaylod is not empty or it is empty, so if statement seems better.

File: main.go (hunk 2 above)
Comment (Misiu) [reply main.go:63]: done :)

File: client.go (hunk 1 above)
Comment (krylovsk) [reply client.go:73]: How about the following?

```go
var payload interface{}
if c.MsgPayload != "" {
	payload = c.MsgPayload
} else {
	payload = make([]byte, c.MsgSize)
}

ch <- &Message{
	Topic:   c.MsgTopic,
	QoS:     c.MsgQoS,
	Payload: payload,
}
}
```
 Sorry for a long turnaround, would you be able to update the PR?

**Review by krylovsk — CHANGES_REQUESTED** [review]

File: client.go (hunk 1 above)
Comment (Misiu) [reply client.go:73]: @krylovsk sorry for the late reply. I'll try to spin the dev environment, but I currently with another project and not sure when I'll find time to properly test this. Can you update the PR?

Comment (krylovsk) [reply client.go]: Hey, just 1.5 years later I managed to get to it  :). The formatting got messy, but I'll fix that in another commit.

**Pull request merged**